packages/nextjs/next.config.js

```diff
@@ -11,4 +13,4 @@ const nextConfig = {
   },
 };
 
-module.exports = nextConfig;
+module.exports = withPlausibleProxy()(nextConfig);
```

Comment:
missing a line here (on save, your IDE should always add a new line at the end of a file)

For VSCode:
![image](https://github.com/scaffold-eth/scaffoldeth.io/assets/2486142/1b2c30b4-9105-4a44-8fe3-9166e243144d)

Reply:
TYSM! Many times I found myself generating differences with that new line at end of file when modifying source code, and never knew if it was my problem or others. Now I know hehe

Thanks for the setup guide, I've just updated it! ♥ Tested it with `next.config.js` and it's working great 🙌